command-line/keyboard-shortcuts.md

```diff
@@ -55,12 +55,12 @@ You don't have to use keyboard shortcuts for everything on your computer, but af
 
 **Mac Utilities (optional):**
 
-* **Alfred 3** - Alfred is an award-winning app for macOS which boosts 
+* **[Alfred 3](https://www.alfredapp.com/?ref=producthunt)** - Alfred is an award-winning app for macOS which boosts 
              your efficiency with hotkeys, keywords, text expansion and more.
              Search your Mac and the web, and be more productive with custom 
              actions to control your Mac.
              
-* **Magnet 2.0** - Window manager for Mac.
+* **[Magnet 2.0](https://magnet.crowdcafe.com/?ref=producthunt)** - Window manager for Mac. Magnet keeps your workspace organized. Activated by dragging, customizable keyboard shortcuts or via menu bar, Magnet declutters your screen by snapping windows into organized tiles.
```

Comment:
Change the link to https://magnet.crowdcafe.com/
And you can start the description like `Magnet is a window manager for Mac`. And after that instead of using Magnet work again and again use `it`.